src/smart-components/OpenChannelSettings/components/ParticipantUI/ParticipantsModal.tsx

```diff
@@ -125,12 +125,22 @@ export default function ParticipantsModal({
                               </MenuItem>
                               <MenuItem
                                 onClick={() => {
-                                  channel?.muteUser(p).then(() => {
-                                    closeDropdown();
-                                  });
+                                  if (p.isMuted) {
+                                    channel?.unmuteUser(p).then(() => {
+                                      closeDropdown();
+                                    });
+                                  } else {
+                                    channel?.muteUser(p).then(() => {
+                                      closeDropdown();
+                                    });
+                                  }
                                 }}
                               >
-                                {stringSet.OPEN_CHANNEL_SETTING__MODERATION__MUTE}
+                                {
+                                  p.isMuted
+                                    ? stringSet.OPEN_CHANNEL_SETTING__MODERATION__UNMUTE
+                                    : stringSet.OPEN_CHANNEL_SETTING__MODERATION__MUTE
+                                }
```

Comment:
So we need to display the items for unmute when it's muted and vice versa, right?

Reply:
Yeah